### ISSUE TITLE
error: cannot find symbol if (tfLite != null) tfLite.setUseNNAPI(isChecked);

### ISSUE THREAD

**Issue by Ajay-Reva** [juandes/mobiledet-tflite-nnapi#1]
symbol:   method setUseNNAPI(boolean)
 location: variable tfLite of type Interpreter

Android SDK 27.0.3, 28.0.3, 29.0.2, 30.0.2
Android NDK 21.0.6113669, 21.3.6528147

**Comment by juandes** [comment]
Hi. Could you please explain the situation better? I can't replicate the problem.

**Comment by Ajay-Reva** [comment]
I tried to build your android project using android studio. 

Android Studio version = 4.0.1
Device API = 28
NDK version = 21.0.6113669

When I build it (green hammer) I see this warning:
```
Warning:root project 'mobiledet-tflite-nnapi': Unable to resolve additional project configuration.
Details: org.apache.tools.ant.BuildException: Basedir /mobiledet-tflite-nnapi/tensorflow-lite does not exist
```

When I ran it 4 days ago I found this error:
`error: cannot find symbol if (tfLite != null) tfLite.setUseNNAPI(isChecked);`

Now it works. I am really sorry, but this is weird. I have tried to work with your repository multiple times before ending up with this same error.
